package internal

func errPanic(err error) {
	panic(err)
}
